src/components/EditFeatureDrawer/EditFeatureFullForm/index.tsx

```diff
@@ -159,6 +161,18 @@ export const EditFeatureFullForm: React.FC<EditFeatureFullFormProps> = ({
             properties[key] = [];
           }
         }
+
+        const isReference = tabConfigs[0].children?.find(cfg => {
```

Comment:
since `isReference` is used only in an if-check, `Array.some()` would be sufficient here

Reply:
```suggestion
        const isReference = tabConfigs[0].children?.some(cfg => {
```

Reply:
You're absolutely right, fixed it.